extensions-core/druid-catalog/src/test/java/org/apache/druid/catalog/sql/CatalogReplaceTest.java

```diff
@@ -92,7 +92,7 @@ public void buildDatasources()
       createTableMetadata(tableBuilder.build());
     });
     DatasourceFacade catalogMetadata =
-        ((DatasourceTable) resolvedTables.get("foo")).effectiveMetadata().catalogMetadata();
```

Comment:
note: why not have a value type of `DatasourceTable` for `RESOLVED_TABLES`?

Reply:
Good question 😅 . Fixed